pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -123,14 +125,15 @@ in mkDerivationWith python3Packages.buildPythonApplication rec {
       --add-flags '--backend ${backend}'
       --set QUTE_QTWEBENGINE_VERSION_OVERRIDE "${lib.getVersion qtwebengine}"
       ${lib.optionalString (pipewireSupport && backend == "webengine") ''--prefix LD_LIBRARY_PATH : ${libPath}''}
+      ${lib.optionalString enableWideVine ''--add-flags "--qt-flag widevine-path=${widevine-cdm}/share/google/chrome/WidevineCdm/_platform_specific/linux_x64/libwidevinecdm.so"''}
     )
   '';
 
   meta = with lib; {
     homepage    = "https://github.com/The-Compiler/qutebrowser";
     description = "Keyboard-focused browser with a minimal GUI";
-    license     = licenses.gpl3Plus;
+    license     = if enableWideVine then licenses.unfree else licenses.gpl3Plus;
```

Comment:
I'm not sure if this is right: qutebrowser is GPL3+ and the fact that you can dynamically load some proprietary object at runtime shouldn't change that.

However you could that this package would be distributing a proprietary library along with qutebrowser: this is not allowed by the GPL unless either it has an exception clause specifically for this library or the library is a "System Library", which is probably not this case.

This sort of mess is why I generally don't want anything to do with proprietary software.

Reply:
It's also possible to argue that the package hasn't changed (and the license should be left as GPL3+) because there's no top-level attribute which provides qutebrowser with the `enableWideVine` flag enabled. Users would have to opt-in and build qutebrowser in this configuration. That makes it the NixOS equivalent of users opting in to install qutebrowser and WideVine on some other distro. 

It also means that the ofborg build probably didn't build the intended package configuration.

Reply:
> Users would have to opt-in and build qutebrowser in this configuration. That makes it the NixOS equivalent of users opting in to install qutebrowser and WideVine on some other distro.

Makes sense to me. So I'd say to keep the license meta as is.

Reply:
I left the GPL3+ license, as proposed.